MinecraftClient/Protocol/Handlers/Protocol16.cs

```diff
@@ -845,5 +845,10 @@ public static bool doPing(string host, int port, ref int protocolversion)
             }
             catch { return false; }
         }
+
+        public bool SelectTrade(int selectedSlot)
+        {
+            return false; //MC 1.13+
```

Comment:
I suspect there is a better way to avoid even needing this in protocol16 file, I can't figure out what part requires this to be in here?